cli/command/trust/inspect.go

```diff
@@ -0,0 +1,97 @@
+package trust
+
+import (
+	"encoding/json"
+	"fmt"
+	"sort"
+
+	"github.com/docker/cli/cli"
+	"github.com/docker/cli/cli/command"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+	"github.com/theupdateframework/notary/tuf/data"
+)
+
+func newInspectCommand(dockerCli command.Cli) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "inspect IMAGE[:TAG] [IMAGE[:TAG]...]",
+		Short: "Return low-level information about keys and signatures",
+		Args:  cli.RequiresMinArgs(1),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return inspectTrustInfo(dockerCli, args)
+		},
+	}
+	return cmd
+}
+
+func inspectTrustInfo(cli command.Cli, remotes []string) error {
+	trustRepoInfoList := []trustRepo{}
+	for _, remote := range remotes {
+		trustInfo, err := getRepoTrustInfo(cli, remote)
+		if err != nil {
+			return err
+		}
+		if trustInfo == nil {
+			continue
+		}
+		trustRepoInfoList = append(trustRepoInfoList, *trustInfo)
+	}
+	trustInspectJSON, err := json.Marshal(trustRepoInfoList)
+	if err != nil {
+		return errors.Wrap(err, "error while serializing trusted repository info")
```

Comment:
Given that JSON is expected, this should still print an empty result (`[]`). For comparison;

```bash
$ docker volume inspect lasdkjadlkjasd
[]
Error: No such volume: lasdkjadlkjasd

$ docker volume inspect lasdkjadlkjasd 2> /dev/null
[]


$ docker trust inspect sdalkjaslkjadlkasjd
No signatures or cannot access sdalkjaslkjadlkasjd

$ docker trust inspect sdalkjaslkjadlkasjd 2> /dev/null
```